carstore/bs.go

```diff
@@ -124,7 +124,7 @@ func (uv *userView) Get(ctx context.Context, k cid.Cid) (blocks.Block, error) {
 		Where("usr = ? AND cid = ?", uv.user, k.String()).
 		First(&info).Error; err != nil {
 		if err == gorm.ErrRecordNotFound {
-			return nil, ipld.ErrNotFound{k}
```

Comment:
I'm really not a fan of forcing keys in cases like this, definitely a style thing but it just feels cleaner not to

Reply:
what about in the case of multiple keys? like:

```
err := shared.TokenPump{TokenSource: dec, TokenSink: enc}.Run()
```
(`cmd/gosky/main.go:722`)